object/syncer_public_api.go

```diff
@@ -66,7 +65,7 @@ func AddUserToOriginalDatabase(user *User) error {
 
 func UpdateUserToOriginalDatabase(user *User) error {
 	syncer, err := getEnabledSyncerForOrganization(user.Owner)
-	if err != nil {
+	if syncer == nil {
```

Comment:
Restore the original error handling:

![image](https://github.com/casdoor/casdoor/assets/3787410/e942329b-785e-4bc6-9f79-b331b6358efa)